Recipes/Recipes.API/Models/Entities/RecipeEntity.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+
+namespace Recipes.API.Models.Entities;
+
+public class RecipeEntity
+{
+    [Key]
+    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+    public int Id { get; set; }
+
+    public int AuthorId { get; set; }
+
+    public string Title { get; set; }
```

Comment:
Maybe worth taking a look at all of the nullability warning reported by the CI. Maybe a `required` would help if that does not break other (e.g. EF Core) stuff

Reply:
added ```required``` keyword or made type nullable where needed

Reply:
I'd make all fields `required` that are non nullable in DB, but what we have currently works as well